### PR TITLE
change inaccurate "RDF" to "RDF/XML" in human-facing serialization lists

### PR DESCRIPTION
**Pull request by TallTed**
in `r2rml_import.vspx`, `rdf_import.vspx`, `dav_browser.xsl`, and `sql_rdf.sql`

### CODE REVIEW

**Comment by TallTed** [comment]
Merge conflicts resolved.

File: binsrc/conductor/r2rml_import.vspx [hunk 1]
```diff
@@ -163,7 +163,7 @@
                           if ((is_ttl + is_xml) = 0)
                           {
                             self.vc_is_valid := 0;
-                            self.vc_error_message := 'You have attempted to upload invalid data. You can only upload RDF, Turtle, N3 serializations of RDF Data to the RDF Data Store.';
+                            self.vc_error_message := 'You have attempted to upload invalid data. You can only upload RDF/XML, Turtle, and N3 serializations of RDF Data to the RDF Data Store through this interface.';
```
Comment (TallTed) [inline binsrc/conductor/r2rml_import.vspx:166]: @pkleef @iv-an-ru -- Should this list also include N-Quads and JSON-LD, as in `rdf_import.vspx`?
```suggestion
                            self.vc_error_message := 'You have attempted to upload invalid data. You can only upload RDF/XML, Turtle, N3, N-Quads, and JSON-LD serializations of RDF Data to the RDF Data Store through this interface.';
```

File: appsrc/ODS-Framework/sql_rdf.sql [hunk 2]
```diff
@@ -94,7 +94,8 @@ create procedure rdf_import (
     if (is_xml = 0 and is_ttl = 0) {
       if (not isnull (pFolder))
         DB.DBA.DAV_DELETE_INT (pFolder, 1, null, null, 0);
-      signal ('ODS10', 'You have attempted to upload invalid data. You can only upload RDF, Turtle, N3 serializations of RDF Data to the RDF Data Store!');
+      signal ('ODS10', 'You have attempted to upload invalid data. You can only upload RDF/XML, Turtle, and N3 serializations of RDF Data to the RDF Data Store through this interface!');
```
Comment (TallTed) [inline appsrc/ODS-Framework/sql_rdf.sql:97]: @pkleef @iv-an-ru -- Should this list also include N-Quads and JSON-LD, as in `rdf_import.vspx`?
```suggestion
      signal ('ODS10', 'You have attempted to upload invalid data. You can only upload RDF/XML, Turtle, N3, N-Quads, and JSON-LD serializations of RDF Data to the RDF Data Store through this interface.');
```

**Comment by TallTed** [comment]
Merge conflicts resolved, again. 

@pkleef @iv-an-ru @openlink -- Please review and merge!

**Comment by pkleef** [comment]
Merged fixes for conductor. 
Rejected fixes for ODS-Framework as this code does not allow these extensions.

File: appsrc/ODS-Framework/sql_rdf.sql (hunk 2 above)
Comment (TallTed) [reply appsrc/ODS-Framework/sql_rdf.sql:97]: ODS-Framework does not allow these serializations.

File: binsrc/conductor/r2rml_import.vspx (hunk 1 above)
Comment (TallTed) [reply binsrc/conductor/r2rml_import.vspx:166]: https://github.com/openlink/virtuoso-opensource/commit/edd6ae6a98d1cb19d42ddc76e320408e5b39b3e1

**Comment by TallTed** [comment]
edd6ae6a98d1cb19d42ddc76e320408e5b39b3e1

**Pull request closed without merging**